doc/user/layouts/shortcodes/rolling-out.html

```diff
@@ -0,0 +1,15 @@
+{{$version := .Get 0}}
+{{$releasePages := where $.Site.RegularPages "Section" "releases"}}
+{{$releasePage := (index (where $releasePages ".File.ContentBaseName" $version) 0)}}
+{{$isReleased := $releasePage.Params.released}}
+{{$releaseDate := $releasePage.Params.date}}
+
+{{if not $isReleased}}
+  <div class="warning">
+    <strong class="gutter">Rolling out</strong>
+    This feature is currently rolling out
+    in <a href="{{$releasePage.RelPermalink}}"><strong>{{$version}}</strong></a>.
+    It may not be available in your region yet.
+    Rollout is scheduled to complete by <strong>{{dateFormat "January 2, 2006" $releaseDate}}</strong>.
```

Comment:
```suggestion
    <strong class="gutter">Unreleased</strong>
    This feature will be released
    in <a href="{{$releasePage.RelPermalink}}"><strong>{{$version}}</strong></a>.
    It may not be available in your region yet.
    The release is scheduled to complete by <strong>{{dateFormat "January 2, 2006" $releaseDate}}</strong>.
```